docs/tools/ColabFold.md

```diff
@@ -70,17 +70,19 @@ The code below is an example of a batch script to run MMSeqs2. The top of the sc
 
 module load uri
```

Comment:
does the `uri` module also require a version now?

Reply:
@cissoucr This should be uri/main now.